### PR TITLE
Makefile added

### PR DESCRIPTION
**Pull request by darecoder**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+IMAGE_PUSH_REGISTRY:=quay.io/emishra/$(EXECUTABLE)
+VERSION:=${shell git describe --tags --always}
+BUILDTIME := ${shell date -u '+%Y-%m-%d_%H:%M:%S'}
+BUILD_DIR:=./bin
+
+.PHONY: build test clean image image-publish
+build: test
+	mkdir -p $(BUILD_DIR)
+	CGO_ENABLED=0 GOOS=linux go build cmd/oc-historical_logs.go
+	chmod +x oc-historical_logs
```
Comment (igor-karpukhin) [inline Makefile]: @darecoder lines 13-14 are better to move to a separate target like `make install`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+test:
+	go test ./pkg/... -cover
+
+test-cover:
+	go test ./pkg/... -coverprofile=coverage.out && go tool cover -html=coverage.out
+
+clean:
+	rm -rf $(BUILD_DIR)/
+
+image: build
```
Comment (igor-karpukhin) [inline Makefile]: I'm not sure if we need an image for this tool

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.7
```
Comment (igor-karpukhin) [inline Dockerfile]: Since we're not planning to use this tool in a container, I'd say we don't need the Dockerfile

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,30 @@
+CONTAINER_ENGINE?=podman
+EXECUTABLE:=log-exploration-oc-plugin
+PACKAGE:=github.com/ViaQ/log-exploration-oc-plugin
+IMAGE_PUSH_REGISTRY:=quay.io/emishra/$(EXECUTABLE)
+VERSION:=${shell git describe --tags --always}
+BUILDTIME := ${shell date -u '+%Y-%m-%d_%H:%M:%S'}
+BUILD_DIR:=./bin
```
Comment (igor-karpukhin) [inline Makefile:4]: `BUILD_DIR` is not used. I guess you wanted to add it to the `build` target? Like `go build -o $(BUILD_DIR)/$(EXECUTABLE) cmd/oc-historical_logs.go`

**Review by igor-karpukhin — CHANGES_REQUESTED** [review]
@darecoder Good job! Please fix some parts I mentioned and we can merge this PR.

File: Makefile (hunk 4 above)
Comment (darecoder) [reply Makefile:4]: Thanks for the review @igor-karpukhin. I'll update this.

File: Makefile (hunk 1 above)
Comment (darecoder) [reply Makefile]: Okay!

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,24 @@
+PACKAGE:=github.com/ViaQ/log-exploration-oc-plugin
+VERSION:=${shell git describe --tags --always}
+BUILDTIME := ${shell date -u '+%Y-%m-%d_%H:%M:%S'}
+BUILD_DIR:=./
```
Comment (igor-karpukhin) [inline Makefile]:
```suggestion
BUILD_DIR:=./bin
```
cc: @darecoder

Comment (igor-karpukhin) [reply Makefile]: Otherwise looks good

**Comment by darecoder** [comment]
Done!

**Review by igor-karpukhin — APPROVED** [review]
Looks good

**Pull request merged**